src/components/Editor/Editor.js

```diff
@@ -605,6 +609,20 @@ class Editor extends React.Component {
                 <FormattedMessage id="saving" defaultMessage="Saving..." />
               </span>
             )}
+            <Form.Item className="Editor__bottom__cancel">
+              {draftId &&
+              <Button
+                type="danger"
+                disabled={loading}
+                onClick={(e) => {
```

Comment:
Can we use `handleCancel` method or something in like this instead? This creates new function on every re-render.